src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.tsx

```diff
@@ -436,10 +432,18 @@ const ADT3DSceneBuilderBase: React.FC<IADT3DSceneBuilderCardProps> = (
             }
 
             if (outlinedElements.length > 0) {
-                setSceneViewAttributes({ outlinedMeshItems: outlinedElements });
+                // See if this makes sense or if it is better to have a setter
+                sceneViewDispatch({
```

Comment:
is it really a reset in this case? I do have some resets that take the value to revert too, so it's really more of a semantic decision than a "properness" question.

Reply:
I would expect it to use the "previously outlined items" ref if it is really a 'reset'

Reply:
Wording here is tricky, I have a SET and RESET action, but I used the set to 'build' the outlined items out of meshIds and colors. I didn't want to make SET more complicated or just create a new reducer action that does the same thing as RESET. Do you think an extra action would be warranted here?